platform/nano/api.go

```diff
@@ -30,17 +29,7 @@ func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
 		return nil, err
 	}
 
-	b, err := json.Marshal(history.History)
```

Comment:
why remove this? `history` field might be empty string (not array)

Reply:
Ahh i see, looks like NANO RPC bug or stupid decision return string when array normally returned of transactions. Logic was returned, i left a comment to avoid such as misunderstanding.